coderd/coderd.go

```diff
@@ -123,20 +125,14 @@ func New(options *Options) *API {
 		options.AgentInactiveDisconnectTimeout = options.AgentConnectionUpdateFrequency * 2
 	}
 	if options.AgentStatsRefreshInterval == 0 {
-		options.AgentStatsRefreshInterval = 10 * time.Minute
+		options.AgentStatsRefreshInterval = 5 * time.Minute
```

Comment:
@coadler looks like this was duplicated below so it was 5 instead of 10, do we want to change it to 10?

Reply:
When I started this PR 1 month ago, both were set to 10. Since one of the values changed I decided to use the updated value. Can revert if needbe.

Reply:
Yeah this is fine, thanks!